src/service/service.ts

```diff
@@ -402,7 +406,7 @@ export class Discv5 extends (EventEmitter as { new (): Discv5EventEmitter }) {
   /**
    * Sends a PING request to a node
    */
-  private sendPing(enr: ENR): void {
+  public sendPing(enr: ENR): void {
```

Comment:
Can you change the param to `remote: ENR | Multiaddr`? Would allow pinging a peer via full multiaddr.